components/button/src/button/button.styles.js

```diff
@@ -32,31 +32,15 @@ export default css`
     }
 
     button:focus {
-        /* Prevent default browser behavior which adds an outline */
-        outline: none;
-    }
-
-    /* Use the ::after pseudo-element for focus styles */
-    button::after {
-        content: ' ';
-        pointer-events: none;
-
-        position: absolute;
-        top: -2px;
-        right: -2px;
-        bottom: -2px;
-        left: -2px;
-
-        border: 2px solid transparent;
-        border-radius: inherit;
-
-        transition: border-color 0.15s cubic-bezier(0.4, 0, 0.6, 1);
+        outline: 3px solid ${theme.focus};
```

Comment:
The table in UX-20 says 2px outline and 2px offset, is the typo in the code?

Reply:
It was a typo in UX-20, but it highlighted an inconsistency that I wasn't happy with. This has been changed and all elements now use the same outline width.